cpu/stm32_common/include/periph_cpu_common.h

```diff
@@ -206,6 +235,37 @@ typedef enum {
 #endif /* ndef CPU_FAM_STM32F1 */
 
 /**
+ * @brief   Override the ADC resolution configuration
+ * @{
```

Comment:
This is not needed and the closing block below as well

Reply:
Or the `@brief` should become `@name`

Reply:
true